site/default.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: " en" }}">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://raw.githubusercontent.com/linz/imagery/master/template/site/css/lui.css">
```

Comment:
hot linking into github is a little risky, I don't think they like people doing this.

Reply:
I'll embed them in the HTML then